addons/binding/org.openhab.binding.sleepiq/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,140 @@
+            <bridge-type-ref id="cloud" />
+        </supported-bridge-type-refs>
+
+        <label>Single Chamber Bed</label>
+        <description>A Sleep Number bed with SleepIQ and a single air chamber</description>
+
+        <channel-groups>
+            <channel-group id="bed" typeId="chamberGroupType"/>
+        </channel-groups>
+
+        <config-description>
+            <parameter name="bedId" type="text">
+                <label>Bed ID</label>
+                <description>The bed identifier identifies a specific bed</description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+    </thing-type>
+    -->
+    
+    <!-- Bed thing with two separate chambers -->
+    <thing-type id="dualBed">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="cloud" />
+        </supported-bridge-type-refs>
+
+        <label>Dual Chamber Bed</label>
+        <description>A Sleep Number bed with SleepIQ and two individual air chambers</description>
+
+        <channel-groups>
+            <channel-group id="left" typeId="chamberGroupType"/>
+            <channel-group id="right" typeId="chamberGroupType"/>
+        </channel-groups>
+
+        <config-description>
+            <parameter name="bedId" type="text">
+                <label>Bed ID</label>
+                <description>The bed identifier identifies a specific bed</description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+    </thing-type>
+    
+    <!-- The attributes of a chamber -->
+    <channel-group-type id="chamberGroupType">
+        <label>Chamber</label>
+        <channels>
+            <channel id="inBed" typeId="inBedType"/>
+            <channel id="sleepNumber" typeId="sleepNumberType"/>
+            <channel id="pressure" typeId="pressureType"/>
+            <channel id="lastLink" typeId="lastLinkType"/>
+            <channel id="alertId" typeId="alertIdType"/>
+            <channel id="alertDetailedMessage" typeId="alertDetailedMessageType"/>
+        </channels>
+    </channel-group-type>
+    
+    <channel-type id="inBedType">
+        <item-type>Switch</item-type>
+        <label>In Bed</label>
+        <description>The presence of a person or object on the chamber</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="sleepNumberType">
+        <item-type>Number</item-type>
+        <label>Sleep Number</label>
+        <description>The Sleep Number setting of the chamber</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="pressureType">
+        <item-type>Number</item-type>
+        <label>Pressure</label>
+        <description>The current pressure inside the chamber</description>
+        <state pattern="%d kPa" readOnly="true" />
+    </channel-type>
+    <channel-type id="lastLinkType">
+        <item-type>String</item-type>
+        <label>Last Link</label>
```

Comment:
The API does call this field "lastLink" and that is why I used the term. It represents a duration of time that has passed since the last communication from the bed to the cloud service. When everything is operating properly, this will be '00:00:00' or will have a small number of seconds (e.g. '00:00:03').